connectors/service_cli.py

```diff
@@ -180,44 +102,42 @@ def run(args):
     # load config
     config = {}
     try:
-        config = load_config(args.config_file)
+        config = load_config(config_file)
         ContentExtraction.set_extraction_config(
             config.get("extraction_service", None)
         )  # Not perfect, let's revisit
     except Exception as e:
         # If something goes wrong while parsing config file, we still want
         # to set up the logger so that Cloud deployments report errors to
         # logs properly
-        set_logger(logging.INFO, filebeat=args.filebeat)
-        logger.exception(f"Could not parse {args.config_file}:\n{e}")
-        raise
+        set_logger(logging.INFO, filebeat=filebeat)
+        msg = f"Could not parse {config_file}. Check logs for more information"
+        logger.exception(f"{msg}.\n{e}")
+        raise ClickException(msg) from e
 
     # Precedence: CLI args >> Config Setting >> INFO
     set_logger(
-        args.log_level or config["service"]["log_level"] or logging.INFO,
-        filebeat=args.filebeat,
+        log_level or config["service"]["log_level"] or logging.INFO,
+        filebeat=filebeat,
     )
 
     # just display the list of connectors
-    if args.action == ["list"]:
+    if action == ("list",):
```

Comment:
Why is it a tuple comparison? Action is also a tuple?

Reply:
Yes it's a tuple:

<img width="170" alt="image" src="https://github.com/elastic/connectors/assets/26525778/cf64cfa6-5fa7-418e-be71-2818410b97ef">